devenv/sync.py

```diff
@@ -128,17 +128,20 @@ def main(context: dict[str, str]) -> int:
         repo,
         reporoot,
         venv_dir,
-        (
-            (
-                "git and precommit",
-                # this can't be done in parallel with python dependencies
-                # as multiple pips cannot act on the same venv
-                ("make", "setup-git"),
-            ),
-        ),
+        (("pre-commit dependencies", ("pre-commit", "install", "--install-hooks", "-f")),),
     ):
         return 1
 
+    git_config = configparser.ConfigParser()
+    git_config.read(f"{reporoot}/.git/config")
+    git_config["blame"] = {"ignoreRevsFile": ".git-blame-ignore-revs"}
+    git_config["branch"] = {"autosetuprebase": "always"}
```

Comment:
I know people won't want this setting always -- I wonder what we can do about that 🤔 

maybe we move that setting to `bootstrap` and then leave it unmanaged afterwards?

Reply:
Mmm, good idea

Reply:
leaning towards putting that in a new `post-bootstrap.py` instead of releasing new devenv just for this change, wdyt?

Reply:
wfm

Reply:
well, will still need a new devenv release for post-bootstrap but that's a one time deal :P

Reply:
https://github.com/getsentry/devenv/pull/126